README.md

```diff
@@ -133,6 +134,8 @@ Both Python components (the logger and the query interface) depend on the follow
 * The [`Pyre base communicator`](https://github.com/ropod-project/ropod_common/tree/master/pyropod/ropod/pyre_communicator) in `ropod_common`
 * `PyYAML`
 * `pymongo`
+* `rosnode`
```

Comment:
`rosnode` is not a package. You can add `rospkg`, and perhaps `catkin-pkg` as well.

Reply:
Resolved [here](https://github.com/ropod-project/black-box/pull/2/commits/f85ad00eb2f41e0a591f74f7f121760bdfce2ffc).